CONTRIBUTING.md

```diff
@@ -1,42 +1,36 @@
 # Before you contribute to the SPDX License List
 Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
 * [License inclusion principles](DOCS/license-inclusion-principles.md) 
+* 
 * [Explanation of the fields contained on the list](DOCS/license-fields.md)
+* 
 * [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List.
+* 
 * [SPDX Specification](https://spdx.org/specifications): It is helpful to be familiar with certain sections of the SPDX Specification thatuse or deal with the SPDX License List. In particular: sub-sections related to license information in Section 3, 4, and 6; Appendices II, IV, and V.
 
-Please join the mailing list and our bi-weekly calls! See details in next section.
+# Join the mailing list and our bi-weekly calls
 
-# How to contribute/participate
 The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
-* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
-* **bi-weekly calls**: used to discuss topics and issues that may be difficult to only discuss via email or via Github. Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
-* **this Github repo**: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes, or other such changes. Requests for new license or exceptions are tracked here, see more info on that process below.
+* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription at [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
```

Comment:
```
-* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription at [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
+* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
```
the old text would render as "at via lists.spdx.org." making the 'at' reduntant.

Reply:
good catch, got it!